code/game/gamemodes/steal_items.dm

```diff
@@ -41,11 +41,6 @@
 	typepath = /obj/item/weapon/hand_tele
 	protected_jobs = list("Captain", "Research Director")
 
-/datum/theft_objective/jetpack
```

Comment:
Told you what to do with this one; could you please make it the Captain's jetpack, instead? Pushing an objective into viability is better than outright removal.